From 69d8bb0054eabdfc299f9fb02c49f2d384f065c7 Mon Sep 17 00:00:00 2001
From: Liu Xinyun <xinyun.liu@intel.com>
Date: Thu, 8 Jul 2010 21:30:05 +0800
Subject: [PATCH] Lower-server-side-GLX-version.patch

 Fix BMC #1384.
 Lower GLX version to 1.2 when use software rendering.

---
 glx/glxdriswrast.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/glx/glxdriswrast.c b/glx/glxdriswrast.c
index 54f4440..845dc4d 100644
--- a/glx/glxdriswrast.c
+++ b/glx/glxdriswrast.c
@@ -510,7 +510,7 @@ __glXDRIscreenProbe(ScreenPtr pScreen)
     __glXScreenInit(&screen->base, pScreen);
 
     screen->base.GLXmajor = 1;
-    screen->base.GLXminor = 4;
+    screen->base.GLXminor = 2;
 
     LogMessage(X_INFO,
 	       "AIGLX: Loaded and initialized %s\n", filename);
-- 
1.7.0.4

